colcon_core/verb/build.py

```diff
@@ -198,7 +198,10 @@ def main(self, *, context):  # noqa: D102
                 ' line:'
                 '\n\t--allow-overriding ' +
                 ' '.join(sorted(override_messages.keys())))
-            raise RuntimeError(override_msg)
+
+            logger.warn(
+                override_msg + '\n\nThis will be promoted to an error in a'
```

Comment:
```suggestion
                override_msg + '\n\nThis may be promoted to an error in a'
```

(Mostly I want to hedge our bets here, since if this becomes a widespread issue that people are complaining about, maybe we won't actually promote it.  But I don't feel strongly about it)